backend/src/map_model.rs

```diff
@@ -53,9 +54,100 @@ pub struct MapModel {
     // TODO Keep edits / state here or not?
     pub undo_stack: Vec<Command>,
     pub redo_queue: Vec<Command>,
-    // Stores boundary polygons in WGS84, with ALL of their GeoJSON props.
-    // TODO Reconsider
-    pub boundaries: BTreeMap<String, Feature>,
+    pub boundaries: BTreeMap<String, NeighbourhoodStats>,
```

Comment:
Note: I'm proposing we no longer tote around these opaque boundary `Feature`'s created on the frontend. 

Instead, that GeoJSON is converted to a NeighbourhoodDefinition struct at the js/wasm boundary, and we only deal with that on the rust side.